Analysis.java

```diff
@@ -0,0 +1,18 @@
+/**
+ * This class makes string-analyzing objects
+ */
+public class Analysis {
+  /**
+   * The method determines whether the order is increasing.
+   * @param orderParts a string we get from (@link Read#readString())
```

Comment:
Who guarantees that orderParts is a string we get from (@link Read#readString())?